### PR TITLE
Adding FireFox Driver setup

### PR DESCRIPTION
**Pull request by nsanku1992**

### CODE REVIEW

File: src/main/java/Demo/DemoTests.java [hunk 1]
```diff
@@ -13,4 +14,10 @@ public ChromeDriver getDriver(){
         WebDriverManager.chromedriver().mac().setup();
         return new ChromeDriver();
     }
+
+    public FirefoxDriver firefoxDriver(){
```
Comment (SainiSachin) [inline src/main/java/Demo/DemoTests.java:18]: Please add documentation for this function.

Comment (SainiSachin) [reply src/main/java/Demo/DemoTests.java:18]: should be : sets and returns the new firefox driver.

File: src/test/java/DemoCore/DemoT.java [hunk 2]
```diff
@@ -11,4 +11,10 @@ public void verifyBonigarciaWorks(){
         WebDriver driver = new DemoTests().getDriver();
         driver.navigate().to("https://mvnrepository.com/artifact/io.github.bonigarcia/webdrivermanager/4.4.3");
     }
+
+    @Test
```
Comment (SainiSachin) [inline src/test/java/DemoCore/DemoT.java:18]: Add documentation for this test method.

**Review by SainiSachin — CHANGES_REQUESTED** [review]
please see my comments

File: src/test/java/DemoCore/DemoT.java (hunk 2 above)
Comment (nsanku1992) [reply src/test/java/DemoCore/DemoT.java:18]: Done

**Pull request merged**